projects/vanilla/src/app/app.module.ts

```diff
@@ -86,12 +91,15 @@ import { SearchFormComponent } from "@sinequa/components/search-form";
 export const startConfig: StartConfig = {
     // app: "training",
     production: environment.production,
-    //autoSAMLProvider: environment.autoSAMLProvider,
+    // autoSAMLProvider: environment.autoSAMLProvider,
     auditEnabled: true
 };
 
 // @sinequa/core config initializer
-export function StartConfigInitializer(startConfigWebService: StartConfigWebService): () => Promise<StartConfig> {
+export function StartConfigInitializer(startConfigWebService: StartConfigWebService, config: GlobalService): () => Promise<StartConfig> {
+    // start listening global configuration changes
+    config.startListening();
```

Comment:
Why not call `startListening()` in the constructor of `GlobalService`? (and if needed add `GlobalService` in the constructor of `AppModule`)

Reply:
The both works, what solution is easy to understand?

Reply:
I think it is a bit confusing to mix the GlobalService and StartConfigService

Reply:
I 'llchange that